test/services/role/services/userPermissions.test.js

```diff
@@ -0,0 +1,144 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userPermissions = app.service('/permissions/user');
+
+describe('userPermissions', async () => {
+	let server;
+	const ROLES = {
+		TEST: 'test',
+		OTHER: 'other',
+		EXTENDED: 'extended',
+		MULTIPLE: 'multiple',
+		XX: 'xx',
+		NOTHING: 'nothing spezial',
+	};
+	const testPermissions = [
+		'SINGING',
+		'DANCE_RAIN',
+		'WALK_LINES',
+		'RUN_FLOOR',
+	];
+
+	const otherPermissions = [
+		'SITTING',
+		'SITTING_ON_CHAIR',
+		'SITTING_ON_DESK',
+	];
+
+	let testRole;
+
+	let testRoleWithDiffrentPermissons;
+
+	let otherRole;
+
+	let testSchool;
+
+	let testSchool2;
+
+	let testUser;
+
+	let testUser2;
+
+	let otherUser;
+
+	let accountTestUser;
+	let accountTestUser2;
+	let accountTestUserOther;
+
+	const commonAccountData = {
+		password: '$2a$10$wMuk7hpjULOEJrTW/CKtU.lIETKa.nEs8fncqLJ74SMeX.fzJACla',
+		activated: true,
+		createdAt: '2017-09-04T12:51:58.49Z',
+	};
+
+	const dataTestUser1 = {
+		username: 'testuser1',
+		...commonAccountData,
+	};
+
+	const dataTestUser2 = {
+		username: 'testuser2',
+		...commonAccountData,
+	};
+
+	const dataTestUserOther = {
+		username: 'testuserother',
+		...commonAccountData,
+	};
+
+	before((done) => {
+		server = app.listen(0, async () => {
+			testRole = await testObjects.createTestRole({
+				name: ROLES.TEST,
+				permissions: testPermissions,
+			});
+
+			testRoleWithDiffrentPermissons = await testObjects.createTestRole({
+				name: ROLES.TEST,
+				permissions: otherPermissions,
+			});
+
+			otherRole = await testObjects.createTestRole({
+				name: ROLES.OTHER,
+				permissions: otherPermissions,
+			});
+
+			testSchool = await testObjects.createTestSchool({
+				permissions: { test: { SITTING:true }},
+			});
+
+			testSchool2 = await testObjects.createTestSchool({
+				permissions: { other: { SITTING:true }},
```

Comment:
Codacy found an issue: [A space is required before '}'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5786177)